topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -320,7 +321,13 @@ private FileWrapper getDescriptorFile(Ga4Ghv20Api ga4Ghv20Api, String trsId, Str
     }
 
     private Optional<FileWrapper> getNextflowMainScript(String nextflowConfigFileContent, Ga4Ghv20Api ga4Ghv20Api, String trsId, String versionId, DescriptorTypeEnum descriptorType) {
-        final String mainScriptPath = NextflowUtilities.grabConfig(nextflowConfigFileContent).getString("manifest.mainScript", "main.nf");
+        final String mainScriptPath;
+        try {
+            mainScriptPath = NextflowUtilities.grabConfig(nextflowConfigFileContent).getString("manifest.mainScript", "main.nf");
```

Comment:
First time I ran this, I believe I got a NextflowParsingException, which caused the program to exit early. Put this in a try catch.

Kathy thinks that it may have just been logging noise that I saw, so I could be wrong. But the method can throw a NextFlowParsingException, so we should catch it anyway.